piksi_tools/console/utils.py

```diff
@@ -125,10 +171,45 @@ def log_time_strings(week, tow):
   return ((t_local_date, t_local_secs), 
           (t_gps_date, t_gps_secs))
 
+
 def call_repeatedly(interval, func, *args):
-    stopped = Event()
-    def loop():
-        while not stopped.wait(interval): # the first call is in `interval` secs
-            func(*args)
-    Thread(target=loop).start()    
-    return stopped.set
+  stopped = Event()
+  def loop():
+      while not stopped.wait(interval): # the first call is in `interval` secs
+          func(*args)
+  Thread(target=loop).start()    
+  return stopped.set
+
+
```

Comment:
Please put this in the gui_utils.py instead of utils file.  The idea is that utils.py has no Traits or UI elements, while gui_utils does.